zuul.d/project-templates.yaml

```
@@ -103,7 +105,9 @@
               - name: github.com/ansible-collections/ansible.netcommon
               - name: github.com/ansible-collections/community.aws
               - name: github.com/ansible-collections/community.general
-        - ansible-test-splitter
+        - ansible-test-splitter:
+            vars:
+              ansible_test_splitter__test_changed: true
```

```suggestion
        - ansible-test-splitter
```

My understanding was that we want `gate` to run the full test suite, while `check` only runs the 'changed' test suite.

---

yes was about to change it too, that was inherited from JillR PR